sources/HUBViewModelBuilderImplementation.m

```diff
@@ -266,6 +266,22 @@ - (void)removeAllComponentModelBuilders
                                                        customData:[self.customData copy]];
 }
 
+#pragma mark - Manipulate custom data
+
+- (void)setCustomDataValue:(nullable id)value forKey:(nonnull NSString *)key
+{
+    NSMutableDictionary *customData = self.customData.mutableCopy;
+    customData = customData ?: [[NSMutableDictionary alloc] init];
```

Comment:
Suggestion: You could avoid `customData` becoming `nullable` by combining this statement with the one above:
```objective-c
NSMutableDictionary * const customData = [self.customData mutableCopy] ?: [NSMutableDictionary new];
```